bundle/QueryType/ParameterProcessor.php

```diff
@@ -99,6 +99,66 @@ static function ($arguments, $name, $default) {
             }
         );
 
+        $expressionLanguage->register(
+            'queryParamString',
+            static function () {},
+            static function ($arguments, $name, $default) {
+                /** @var \Symfony\Component\HttpFoundation\Request $request */
+                $request = $arguments['request'];
+
+                if ($request->query->has($name)) {
+                    return (string)$request->query->get($name, $default);
+                }
+
+                return $default;
+            }
+        );
+
+        $expressionLanguage->register(
+            'queryParamInt',
+            static function () {},
+            static function ($arguments, $name, $default) {
+                /** @var \Symfony\Component\HttpFoundation\Request $request */
+                $request = $arguments['request'];
+
+                if ($request->query->has($name)) {
+                    return (int)$request->query->get($name, $default);
+                }
+
+                return $default;
+            }
+        );
+
+        $expressionLanguage->register(
+            'queryParamFloat',
+            static function () {},
+            static function ($arguments, $name, $default) {
+                /** @var \Symfony\Component\HttpFoundation\Request $request */
+                $request = $arguments['request'];
+
+                if ($request->query->has($name)) {
+                    return (float)$request->query->get($name, $default);
+                }
+
+                return $default;
+            }
+        );
+
+        $expressionLanguage->register(
+            'queryParamBool',
+            static function () {},
+            static function ($arguments, $name, $default) {
+                /** @var \Symfony\Component\HttpFoundation\Request $request */
+                $request = $arguments['request'];
+
+                if ($request->query->has($name)) {
+                    return (bool)$request->query->get($name, $default);
```

Comment:
You can use `$request->query->getBoolean()`

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/97/commits/f72f5ecae52253ce294122b73d769c9a0157745a